website/static/js/oauthAddonNodeConfig.js

```diff
@@ -0,0 +1,244 @@
+/**
+ * Module that controls the addon node settings. Includes Knockout view-model
+ * for syncing data, and HGrid-folderpicker for selecting a folder.
+ */
+'use strict';
+
+var ko = require('knockout');
+require('knockout.punches');
```

Comment:
There are four uses, three of which come from things that aren't necessarily available / are dependent on user input. Also, knockout punches is used in enough other places that it's probably cached in the browser by the time a user hits this page.